core/src/main/java/org/opensearch/sql/planner/physical/collector/Rounding.java

```diff
@@ -46,13 +46,13 @@ public static Rounding<?> createRounding(SpanExpression span) {
     if (DOUBLE.isCompatible(type)) {
       return new DoubleRounding(interval);
     }
-    if (type.equals(TIMESTAMP)) {
+    if (type.equals(TIMESTAMP) || type.typeName().equalsIgnoreCase(TIMESTAMP.typeName())) {
```

Comment:
what case covered in here?

Reply:
`type` could be an `OpenSearchDateType`, rather `ExprCoreType` after #2762